Tests/Data/HydroMechanics/StaggeredScheme/InjectionProduction1D/InjectionProduction1D.prj

```diff
@@ -71,6 +76,18 @@
                     </properties>
                 </phase>
             </phases>
+            <properties>
+                <property>
+                    <name>reference_temperature</name>
+                    <type>Constant</type>
+                    <value>295.15</value>
```

Comment:
It's not 293.15K ?

Reply:
The reference_temp. is in K, so that means 20°C what's wring with this?